eng/pipelines/templates/jobs/analyze.yml

```diff
@@ -50,3 +50,53 @@ jobs:
       }
     displayName: 'Validate Package README.md files'
     condition: succeededOrFailed()
+
+  - task: Powershell@2
+    displayName: "Build Azure MCP Server for metadata verification"
+    inputs:
+      pwsh: true
+      workingDirectory: $(Build.SourcesDirectory)
+      targetType: inline
+      script: |
+        Write-Host "Building Azure.Mcp.Server for metadata verification..."
+        dotnet build "$(Build.SourcesDirectory)/servers/Azure.Mcp.Server/src/Azure.Mcp.Server.csproj" -c Debug
+        if ($LASTEXITCODE -ne 0) {
+          Write-Error "Build failed"
+          exit $LASTEXITCODE
+        }
+        
+        # Determine the correct executable name based on OS
```

Comment:
This should not be hard-coded to azmcp because we need this to work for all the servers.